### ISSUE TITLE
CC(2.12-beta) on Linux can't open `ply` file.

### ISSUE THREAD

**Issue by hiroyuki-sato** [CloudCompare/CloudCompare#1591]
Hello, members. For investigating #1590, I also try to use CC on Linux.
And I found another issue.  Does this behavior expect?

**Describe the bug**

CC(2.12-beta) can't open `ply` file. (CC 2.11.3 open it within a second).

**How to reproduce**

1. Open Cloud compare. (It installed using `flatpak`: `flatpak install flathub org.cloudcompare.CloudCompare`
2. Open the attached file ([test.ply.gz](https://github.com/CloudCompare/CloudCompare/files/7963694/test.ply.gz))
3. CC show `Hum, do you really expect 17529776 point clouds?`. Press YES.
4. It shows `open bin file (old style)` and takes a long time. (over five minutes).
5. Finally, It output `An error occurred while loading 'test': see console`. Console says (`Exception: std::bad_alloc`)

(I open the same file with CC 2.11(I installed it using `snap install cloudcompare`). It takes less than 1sec).

**Expected behaviour**

Open `ply` file immediately.

**Additional context**

**Your environment**
 - CC Version (or git hash/tag): 2.12-beta
 - OS & Version: Ubuntu 20.04 (Intel)
 - Graphics card: built-in Intel graphic.
 - Qt Version (if compiling): 

best regards.

**Comment by tmontaigu** [comment]
Probably same problem as in https://github.com/CloudCompare/CloudCompare/issues/1574 your flatpak (the command, not the Cloudcompare flatpak) is probably outdated

**Comment by hiroyuki-sato** [comment]
Hello, @tmontaigu. Thank you for your comment. 

* I installed the latest CC from the source. It worked fine on Ubuntu 12.04. (opened `ply` file quickly).
* I installed flatpak 1.12.4 with the following [Ubuntu Quick setup](https://flatpak.org/setup/Ubuntu) instruction. (`apt install flatpak` installed 1.6.5)

Does this mean that flatpack built CloudCompare with an outdated library?

Anyway, This issue resolve.  I'll close this issue after your comment.

```
      $ sudo add-apt-repository ppa:flatpak/stable
      $ sudo apt update
      $ sudo apt install flatpak
```
